service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -153,10 +153,11 @@ private List<String> getWorkflowErrors(Run updatableRun) {
       }
     } catch (Exception e) {
       String errorMessage =
-          "Error fetching Cromwell-level error from Cromwell for run %s."
-              .formatted(updatableRun.runId());
+          "Error fetching Cromwell-level error from Cromwell for workflow %s."
```

Comment:
Since this is already attached to a run record, do we really need the workflow ID in the message itself?

Reply:
TOL: Should we make the error message DRY? It seems we are repeating Cromwell here. Maybe something like this?
```Error fetching Cromwell-level error for workflow %s.```